tests/app/service/test_send_one_off_notification.py

```diff
@@ -79,16 +88,17 @@ def test_send_one_off_notification_calls_persist_correctly(
         notification_type=SMS_TYPE,
         api_key_id=None,
         key_type=KEY_TYPE_NORMAL,
-        created_by_id=str(service.created_by_id)
+        created_by_id=str(service.created_by_id),
+        reply_to_text='testing'
```

Comment:
I can't see where this is set - it would make sense if this was 'sms_reply_to_text for clarity'

Reply:
'testing' is the default sms sender for the service, this is checking the default is used.